PowerEditor/installer/nativeLang/french.xml

```diff
@@ -21,16 +21,16 @@ Translation note:
 					<Item menuId="macro" name="&amp;Macro"/>
 					<Item menuId="run" name="E&amp;xécution"/>
 					<Item menuId="Plugins" name="Modules d’ex&amp;tension"/>
-					<Item menuId="Window" name="&amp;Documents"/>
+					<Item menuId="Window" name="&amp;Onglets ouverts"/>
 				</Entries>
 				<!-- Sub Menu Entries -->
 				<SubEntries>
-					<Item subMenuId="file-openFolder" name="Ouvrir le répertoire du fichier en cours"/>
-					<Item subMenuId="file-closeMore" name="Fermer plusieurs documents"/>
+					<Item subMenuId="file-openFolder" name="Utiliser le chemin du fichier en cours et :"/>
```

Comment:
> I would say: "Utiliser le chemin du fichier en cours pour ouvrir", but it's too long. Let's add "dans" in the original text so it'll be: "Ouvrir le répertoire du fichier en cours avec"

"Utiliser le chemin du fichier en cours pour ouvrir" is indeed a bit long but it better prepares for the choices that follow. The term "open" being common can be repeated in the three following choices allows to better respect the French language. I propose: "Utiliser le chemin du fichier en cours et" because it is indeed the path that we use for the following and not the directory itself.

Reply:
"Ouvrir le répertoire du fichier en cours avec" is followed by the commands "Explorer", "cmd" & "Folder as Workspace".
We open indeed the current file directory into the 3 commands.